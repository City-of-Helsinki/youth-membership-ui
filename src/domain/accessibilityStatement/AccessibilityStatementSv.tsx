import React, { Fragment } from 'react';

import Text from '../../common/components/text/Text';

function AccessibilityStatementSv() {
  return (
    <Fragment>
      <Text variant="h1">Tillgänglighetsutlåtande</Text>
      <p>
        Detta tillgänglighetsutlåtande gäller Helsingfors stads webbplats
        Ungdomstjänsternas medlem ansökan . Webbplatsens adress är
        https://jassari.hel.fi.
      </p>

      <Text variant="h2">Lagbestämmelser som gäller webbplatsen</Text>
      <p>
        Denna webbplats har offentliggjorts efter 23.9.2018. Webbplatsen ska
        uppfylla lagens krav på tillgänglighet.
      </p>

      <Text variant="h2">Stadens mål</Text>
      <p>
        När det gäller tillgänglighet till digitala tjänster har Helsingfors
        stad som mål att uppnå minst nivå AA eller bättre enligt
        WCAG-anvisningarna, om det är rimligt.
      </p>

      <Text variant="h2">Fullgörandestatus</Text>
      <p>
        Denna webbplats förenlighet med tillgänglighetskraven är inte ännu
        bedömd.
      </p>

      <Text variant="h2">Utarbetande av tillgänglighetsutlåtande</Text>
      <p>Detta utlåtande har upprättats 6.4.2020.</p>

      <Text variant="h2">Uppdatering av tillgänglighetsutlåtande</Text>
      <p>
        Tillgänglighetsutlåtandet uppdateras till att motsvara de observationer
        som gjorts om överensstämmelse med tillgänglighetskraven efter att en
        granskning genomförts.
      </p>

      <Text variant="h2">Begäran om uppgifter i tillgänglig form</Text>
      <p>
        Om användaren inte upplever sig få webbplatsens innehåll i tillgänglig
        form, kan användaren begära dessa uppgifter per e-post{' '}
        <a href="mailto:helsinki.palaute@hel.fi">helsinki.palaute@hel.fi</a>{' '}
        eller med responsformulär på{' '}
        <a href="https://www.hel.fi/helsinki/sv/stad-och-forvaltning/delta/feedback">
          www.hel.fi/feedback
        </a>
        . Strävan är att svara på förfrågan inom rimlig tid.{' '}
      </p>

      <Text variant="h2">Återkoppling och kontaktuppgifter</Text>
      <p>
        Helsingfors stad, Kultur och fritid
        <br />
        Ungdomstjänster
        <br />
        <a href="mailto:nuorisopalvelut@hel.fi">nuorisopalvelut@hel.fi</a>
      </p>

      <Text variant="h2">Helsingfors stad och tillgänglighet</Text>
      <p>
        Helsingfors stad har som mål att vara en tillgänglig stad för alla.
        Stadens mål är att det ska vara så lätt som möjligt för alla stadsbor
        att röra sig och verka i Helsingfors och att alla innehåll och tjänster
        ska vara tillgängliga för alla.{' '}
      </p>
      <p>
        Staden främjar tillgängligheten för digitala tjänster genom att
        förenhetliga publiceringsarbetet och ordna utbildning om tillgänglighet
        för sin personal.{' '}
      </p>
      <p>
        Tillgänglighetsnivån för webbplatser följs upp kontinuerligt när
        webbplatserna underhålls. Observerade brister hanteras omedelbart. Vår
        strävan är att genomföra nödvändiga ändringar så snabbt som möjligt.
      </p>

      <Text variant="h2">Handikappade och hjälpmedelsanvändare</Text>
      <p>
        Staden erbjuder rådgivning och stöd för handikappade och
        hjälpmedelsanvändare. Stöd kan fås på de rådgivningssidor som anges på
        stadens sidor och på telefonrådgivningen.{' '}
      </p>

      <Text variant="h2">Godkännande av tillgänglighetsutlåtande</Text>
      <p>Detta utlåtande har godkänts 17.1.2020</p>
      <p>
        Kultur och fritid,
        <br />
        Helsingfors stad
      </p>
    </Fragment>
  );
}

export default AccessibilityStatementSv;
