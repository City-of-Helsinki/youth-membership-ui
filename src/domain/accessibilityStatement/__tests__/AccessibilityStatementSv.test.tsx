import React from 'react';
import { render } from '@testing-library/react';

import AccessibilityStatementSv from '../AccessibilityStatementSv';

test('matches snapshot', () => {
  const wrapper = render(<AccessibilityStatementSv />);
  expect(wrapper.container).toMatchSnapshot();
});
