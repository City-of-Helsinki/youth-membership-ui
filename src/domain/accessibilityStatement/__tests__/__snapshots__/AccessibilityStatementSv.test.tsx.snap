// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <h1
    class="text h1 "
  >
    Tillgänglighetsutlåtande
  </h1>
  <p>
    Detta tillgänglighetsutlåtande gäller Helsingfors stads webbplats Ungdomstjänsternas medlem ansökan . Webbplatsens adress är https://jassari.hel.fi.
  </p>
  <h2
    class="text h2 "
  >
    Lagbestämmelser som gäller webbplatsen
  </h2>
  <p>
    Denna webbplats har offentliggjorts efter 23.9.2018. Webbplatsen ska uppfylla lagens krav på tillgänglighet.
  </p>
  <h2
    class="text h2 "
  >
    Stadens mål
  </h2>
  <p>
    När det gäller tillgänglighet till digitala tjänster har Helsingfors stad som mål att uppnå minst nivå AA eller bättre enligt WCAG-anvisningarna, om det är rimligt.
  </p>
  <h2
    class="text h2 "
  >
    Fullgörandestatus
  </h2>
  <p>
    Denna webbplats förenlighet med tillgänglighetskraven är inte ännu bedömd.
  </p>
  <h2
    class="text h2 "
  >
    Utarbetande av tillgänglighetsutlåtande
  </h2>
  <p>
    Detta utlåtande har upprättats 6.4.2020.
  </p>
  <h2
    class="text h2 "
  >
    Uppdatering av tillgänglighetsutlåtande
  </h2>
  <p>
    Tillgänglighetsutlåtandet uppdateras till att motsvara de observationer som gjorts om överensstämmelse med tillgänglighetskraven efter att en granskning genomförts.
  </p>
  <h2
    class="text h2 "
  >
    Begäran om uppgifter i tillgänglig form
  </h2>
  <p>
    Om användaren inte upplever sig få webbplatsens innehåll i tillgänglig form, kan användaren begära dessa uppgifter per e-post
     
    <a
      href="mailto:helsinki.palaute@hel.fi"
    >
      helsinki.palaute@hel.fi
    </a>
     
    eller med responsformulär på
     
    <a
      href="https://www.hel.fi/helsinki/sv/stad-och-forvaltning/delta/feedback"
    >
      www.hel.fi/feedback
    </a>
    . Strävan är att svara på förfrågan inom rimlig tid.
     
  </p>
  <h2
    class="text h2 "
  >
    Återkoppling och kontaktuppgifter
  </h2>
  <p>
    Helsingfors stad, Kultur och fritid
    <br />
    Ungdomstjänster
    <br />
    <a
      href="mailto:nuorisopalvelut@hel.fi"
    >
      nuorisopalvelut@hel.fi
    </a>
  </p>
  <h2
    class="text h2 "
  >
    Helsingfors stad och tillgänglighet
  </h2>
  <p>
    Helsingfors stad har som mål att vara en tillgänglig stad för alla. Stadens mål är att det ska vara så lätt som möjligt för alla stadsbor att röra sig och verka i Helsingfors och att alla innehåll och tjänster ska vara tillgängliga för alla.
     
  </p>
  <p>
    Staden främjar tillgängligheten för digitala tjänster genom att förenhetliga publiceringsarbetet och ordna utbildning om tillgänglighet för sin personal.
     
  </p>
  <p>
    Tillgänglighetsnivån för webbplatser följs upp kontinuerligt när webbplatserna underhålls. Observerade brister hanteras omedelbart. Vår strävan är att genomföra nödvändiga ändringar så snabbt som möjligt.
  </p>
  <h2
    class="text h2 "
  >
    Handikappade och hjälpmedelsanvändare
  </h2>
  <p>
    Staden erbjuder rådgivning och stöd för handikappade och hjälpmedelsanvändare. Stöd kan fås på de rådgivningssidor som anges på stadens sidor och på telefonrådgivningen.
     
  </p>
  <h2
    class="text h2 "
  >
    Godkännande av tillgänglighetsutlåtande
  </h2>
  <p>
    Detta utlåtande har godkänts 17.1.2020
  </p>
  <p>
    Kultur och fritid,
    <br />
    Helsingfors stad
  </p>
</div>
`;
